front/src/common/api/osrdEditoastApi.ts

```diff
@@ -1895,68 +1883,84 @@ export type RollingStockUsage = {
     train_schedule_id: number;
   };
 };
-export type SearchTrackResult = {
+export type SearchResultItemTrack = {
   infra_id: number;
   line_code: number;
   line_name: string;
 };
-export type SearchOperationalPointResult = {
+export type GeoJsonPointValue = number[];
+export type GeoJsonPoint = {
+  coordinates: GeoJsonPointValue;
+  type: 'Point';
+};
+export type SearchResultItemOperationalPoint = {
   ch: string;
-  geographic: Point;
-  infra_id?: string;
+  geographic: GeoJsonPoint;
+  infra_id: number;
   name: string;
   obj_id: string;
-  schematic: Point;
+  schematic: GeoJsonPoint;
   track_sections: {
     position: number;
     track: string;
   }[];
   trigram: string;
-  uic?: number;
+  uic: number;
 };
-export type SearchSignalResult = {
-  geographic: Point;
+export type SearchResultItemSignal = {
+  geographic: GeoJsonPoint;
   infra_id: number;
   label: string;
-  line_code: string;
+  line_code: number;
   line_name: string;
-  schematic: Point;
+  schematic: GeoJsonPoint;
   settings: string[];
   signaling_systems: string[];
   sprite?: string | null;
   sprite_signaling_system?: string | null;
 };
-export type SearchStudyResult = {
-  description?: string;
+export type SearchResultItemProject = {
+  description: string;
   id: number;
+  image: number;
```

Comment:
I'm sure image can be null/undefined, and I believe that description, studies_count and tags can be too... Can we check with editoast model ?
```suggestion
  image?: number;
```